packages/assets-controllers/src/AccountTrackerController.test.ts

```diff
@@ -50,31 +59,42 @@ describe('AccountTrackerController', () => {
 
   it('should get real balance', async () => {
     const address = '0xc38bf1ad06ef69f0c04e29dbeb4152b4175f0a8d';
+
+    mockedQuery.mockReturnValueOnce(Promise.resolve('0x10'));
+
     const controller = new AccountTrackerController(
       {
         onPreferencesStateChange: sinon.stub(),
         getIdentities: () => {
           return { [address]: {} as ContactEntry };
         },
+        getSelectedAddress: () => '0x0',
+        getMultiAccountBalancesEnabled: () => true,
       },
       { provider },
     );
+
     await controller.refresh();
+
     expect(controller.state.accounts[address].balance).toBeDefined();
+    expect(controller.state.accounts[address].balance).toBe('0x10');
   });
 
   it('should sync balance with addresses', async () => {
     const address = '0xc38bf1ad06ef69f0c04e29dbeb4152b4175f0a8d';
+
     const controller = new AccountTrackerController(
       {
         onPreferencesStateChange: sinon.stub(),
         getIdentities: () => {
           return {};
         },
+        getSelectedAddress: () => '0x0',
```

Comment:
Q: if getIdentities returns empty, does it mean we can't have a selected address?